src/index.html

```diff
@@ -21,7 +21,370 @@
     />
```

Comment:
You are trying to link a SCSS file to your HTML. However, browsers don't understand SCSS, they only understand CSS. You need to compile your SCSS to CSS and link the CSS file here.